hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1177,7 +1227,8 @@ public String listStatus(final Path path, final String startFrom,
                           blockSize,
                           lastModifiedMillis,
                           entryPath,
-                          entry.eTag()));
+                          entry.eTag(),
+                          encryptionContext));
```

Comment:
Inline with https://github.com/apache/hadoop/pull/3440#discussion_r989075168. Need to have encryptionContext in VersionedFileStatus object.